R-package/demo/multiclass.R

```diff
@@ -34,18 +38,20 @@ model <- lgb.train(
 my_preds <- predict(model, test[, 1L:4L])
 
 # Method 2 of training, identical
-model <- lgb.train(
-    list()
-    , dtrain
-    , 100L
-    , valids
-    , min_data = 1L
+params <- list(
+    min_data = 1L
     , learning_rate = 1.0
     , early_stopping_rounds = 10L
     , objective = "multiclass"
     , metric = "multi_error"
     , num_class = 3L
 )
+model <- lgb.train(
+    params
+    , dtrain
+    , 100L
+    , valids
```

Comment:
Diff of this suggestion looks terrible, but it just proposes moving `early_stopping_rounds` from `params` to `train()` according to all other demos.
```suggestion
    , objective = "multiclass"
    , metric = "multi_error"
    , num_class = 3L
)
model <- lgb.train(
    params
    , dtrain
    , 100L
    , valids
    , early_stopping_rounds = 10L
```